src/object-properties.js

```diff
@@ -23,7 +25,8 @@ const properties = {
    */
   footnote: '',
   /**
-   * @type {'regular'|'scroll'|'free'}
+   * How the org chart is navigated
```

Comment:
This is a spec change and is best to do in separate PR. It also looks like a breaking change?

Reply:
It's been this way in the code for the whole time, just documented wrong, but it should be in a different PR nevertheless 👍

Reply:
Just remember that it will be breaking change from API Gov perspective, so once you do a new release make sure to those spec changes approved.